open-apparel-registry/open-apparel-registry
# Ignore duplicate rows

## Overview

If a list contains duplicate rows, previously both rows were fully
processed. In cases where the duplicate item created a new facility,
this new facility would then end up being created twice, resulting
in duplicate facilities.

We now mark duplicate FacilityListItems from the same list as DUPLICATE,
preventing the further processing of the second item.

Connects #2021 

## Demo

<img width="1436" alt="Screen Shot 2022-08-10 at 4 39 15 PM" src="https://user-images.githubusercontent.com/21046714/184015715-600ee769-4f14-4956-9d5a-06f7b1e40ec4.png">

## Testing Instructions

* Submit and fully process a list with duplicate rows:
[Duplicates.csv](https://github.com/open-apparel-registry/open-apparel-registry/files/9303889/Duplicates.csv)
* The list should have two errors (duplicates) and five successes.
* View the processed lists under [My Lists](http://localhost:6543/lists). Two items should be marked DUPLICATE. 

## Checklist
